tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -58,16 +62,26 @@ module('Acceptance | guid file | registration files', hooks => {
         assert.dom('[data-test-tags-button]').exists('Tags button exists');
         await percySnapshot('Acceptance | guid file | registration files | mobile view | file renderer');
 
+        assert.dom('[data-test-file-renderer]').exists();
+        assert.dom('[data-test-metadata-tab]').doesNotExist();
+        assert.dom('[data-test-revisions-tab]').doesNotExist();
+        assert.dom('[data-test-tags-tab]').doesNotExist();
+
+        // click tags and verify appropriate panels shown
         await click('[data-test-versions-button]');
         assert.dom('[data-test-revisions-tab]').exists('Revisions shown');
-        assert.dom('[data-test-file-renderer]').doesNotExist('File renderer is hidden');
+        assert.dom('[data-test-file-renderer]').isNotVisible('File renderer is hidden');
```

Comment:
I thought we decided to use `.doesNotExist` rather than `.isNotVisible` unless `.doesNotExist` just doesn't work. Here and below.

Reply:
This is now updated for this test as well.